pyClient/zeth/joinsplit.py

```diff
@@ -184,8 +189,7 @@ def compute_nullifier(
     binary_ask = digest_to_binary_string(spending_authority_ask)
     first_252bits_ask = binary_ask[:252]
     left_leg_bin = "1110" + first_252bits_ask
-    left_leg = int(left_leg_bin, 2).to_bytes(32, byteorder='little')
-
+    left_leg = int(left_leg_bin, 2).to_bytes(32, byteorder='big')
```

Comment:
how did you come up in this change ? you checked if the value corresponded to the primary inputs ?

Reply:
This came from a previous change I made (using bytes directly instead of converting back and forth from strings).  It only caused an error when the contract started checking the value as used in hsig.  I checked this against the original binary str -> int -> hex -> bytes code and realised it must end up big endian.
(As you noted in a discussion offline, the value get's recomputed by the prover server, which is why this error was not flagged until the h_sig check was made in the contract)